systems/robotInterfaces/QPLocomotionPlan.m

```diff
@@ -172,17 +167,17 @@
           if obj.toe_off_active.(foot_name)
             body_mask = obj.supports(supp_idx).bodies == body_id;
             if ~isempty(obj.supports(supp_idx).contact_groups{body_mask})
-              obj.supports(supp_idx) = obj.supports(supp_idx).setContactPts(body_mask, rpc.contact_groups{body_id}.toe, {'toe'});
+              obj.supports(supp_idx) = obj.supports(supp_idx).setContactPts(body_mask, obj.robot_property_cache.contact_groups{body_id}.toe, {'toe'});
             end
             qp_input.joint_pd_override(end+1) = struct('position_ind', kny_ind,...
-                                                       'qi_des', obj.MIN_KNEE_ANGLE,...
-                                                       'qdi_des', 0,...
-                                                       'kp', obj.KNEE_KP,...
-                                                       'kd', obj.KNEE_KD,...
-                                                       'weight', obj.KNEE_WEIGHT);
-            if obj.body_motions(j).toe_off_allowed(body_t_ind)
-              obj = obj.updateSwingTrajectory(t_plan, j, body_t_ind, kinsol, qd);
-            end
+              'qi_des', obj.MIN_KNEE_ANGLE,...
```

Comment:
This whole file is now deprecated (replaced by the C++ version and the Matlab wrapper). I just discussed with @rdeits, and we decided we're going to get rid of it in this pull request rather than cleaning it up later to avoid confusion like this.